cloudsql/README.md

```diff
@@ -0,0 +1,56 @@
+## CloudSQL Example
+
+This example shows how to build an application that uses [Google Cloud SQL](https://cloud.google.com/sql/docs/introduction)
+using Kubernetes and [Docker](https://www.docker.com/).
+
+The example consists of a single pod containing two containers:
+
+- A web frontend container running Wordpress.
+- A [Cloud SQL Proxy](https://github.com/GoogleCloudPlatform/cloudsql-proxy/) container providing connectivity to Cloud SQL.
+
+### Prerequisites
+
+This example requires a running Kubernetes cluster at version 1.2 or higher. See the [Getting Started guides](../../docs/getting-started-guides/)
+for how to get started. And follow the [Prerequisites](../../docs/user-guide/prereqs.md) to make sure your `kubectl` is ok. As noted above, if
```

Comment:
These doc links are broken. For Container Engine samples, let's link directly to cloud.google.com. We don't need to have as big of a prereq section, because the sample is for Container Engine, and not all possible Kubernetes deployments.